cli/src/app.rs

```diff
@@ -81,8 +85,15 @@ impl App {
                     String::from("not installed")
                 }
             );
+
+            if release.prerelease {
+                println!("{release_title} [{}]", "prerelease".red().to_string());
```

Comment:
Maybe `orange` color would be better since `red` is usually a sign of something negative/erroneous. What do you think?

Reply:
yeah i agree. I wanted the user to be cautious that the build was prerelase and red stood out the most to me. Let me play with some colors and then post it here